components/common/Banner/StudyBanner/index.tsx

```diff
@@ -0,0 +1,52 @@
+import styled from '@emotion/styled';
+import dynamic from 'next/dynamic';
+import Link from 'next/link';
+
+import Responsive from '@/components/common/Responsive';
+import { textStyles } from '@/styles/typography';
+
+import { CLOSE_DATE, CREW_LINK, OPEN_DATE, TERM } from './contants';
+import MobileStudyBanner from './MobileStudyBanner';
+const CountdownTimer = dynamic(() => import('@/components/common/Banner/StudyBanner/CountdownTimer'), {
+  ssr: false,
+});
+
+interface StudyBannerProps {
+  className?: string;
+}
+
+export default function StudyBanner({ className }: StudyBannerProps) {
+  return (
+    <>
+      <Responsive only='mobile' asChild>
+        <Link href={CREW_LINK} className={className}>
```

Comment:
p1; 링크를 `constants/links` 의 playgroundLink를 사용해서 넣어주세요!

```suggestion
        <Link href={playgroundLink.groupList()} className={className}>
```

Reply:
이런게 있었군여어!